attributes/default.rb

```diff
@@ -4,12 +4,8 @@
 default['omnibus']['build_user_password'] = "#{ENV['BUILD_USER_PASSWORD']}"
 
 default['dd-agent-builder']['install_dir'] = 'C:\opt\datadog-agent'
-default['dd-agent-builder']['dd-agent_branch'] = ENV['DD_AGENT_BRANCH'] || 'master'
 default['dd-agent-builder']['dd-agent-omnibus_branch'] = ENV['DD_AGENT_OMNIBUS_BRANCH'] || 'master'
-default['dd-agent-builder']['omnibus-software_branch'] = ENV['OMNIBUS_SOFTWARE_BRANCH'] || 'master'
-default['dd-agent-builder']['omnibus-ruby_branch'] = ENV['OMNIBUS_RUBY_BRANCH'] || 'datadog-5.5.0'
-default['dd-agent-builder']['integrations-core_branch'] = ENV['INTEGRATIONS_CORE_BRANCH'] || 'master'
-default['dd-agent-builder']['trace-agent_branch'] = ENV['TRACE_AGENT_BRANCH'] || 'master'
+default['dd-agent-builder']['release_version'] = ENV["RELEASE_VERSION"]
```

Comment:
I think we should set `['dd-agent-builder']['release_version']` to `nightly` in `attributes.json` so that builds on `dd-agent-builder/master` have a valid release version and can succeed. What do you think?

Reply:
Good idea !